src/main/resources/application.yml

```diff
@@ -70,6 +70,21 @@ workspace:
     uri: ${env.db.host}/${env.db.ws.name}
     username: ${env.db.ws.user}
 
+  # Local servers will point to Tools RBS by default.
+  # These values may be overridden by Helm.
+  buffer:
+    enabled: true
+    client-credential-file-path: rendered/buffer-client-sa-account.json
+    instanceUrl: https://buffer.tools.integ.envs.broadinstitute.org
+    poolId: workspace_manager_v6
+
+  spend:
+    spend-profiles:
+    - # A default spend profile available in dev Sam.
+      id: wm-default-spend-profile
+      # The billing account workspace-dev has access to.
+      billing-account-id: 01A82E-CA8A14-367457
```

Comment:
Hmmm, this does mean the CWB deployment gets these values by default. Does our helm setup necessarily override them, or only if a helm value is specified.
I'm not convinced we shouldn't instead have a non-CWB spring profile to encode these kinds of things. Then make it easy from bootRun or a helper script to define whether we want a non-CWB version running. WDYT?
Either way, check the src/test resources for applications that are rendered obsolete by this.

Reply:
(Is there an accepted term for CWB vs non-CWB/cloud contexts? Would be nice to improve our terminology here and even record it in some documentation)

Reply:
[Helm explicitly overrides](https://github.com/broadinstitute/terra-helm/blob/master/charts/workspacemanager/values.yaml#L192) `workspace.buffer.enabled` with a default value of `false` for all environments that don't set it otherwise. The other values will now be set in CWB, but they aren't used unless `enabled` is true.

My concern with splitting this into a separate spring profile is that we're introducing another configuration mechanism without really saving any complexity. We wouldn't just have one CWB and one non-CWB version, we would still need to pass through all the environment-specific configuration - except it would be across two application files instead of one, and we would need to set another env variable everywhere we call bootRun. Spend profiles and RBS are also currently the only difference between the two - a separate profile makes more sense to me if these diverge more, but I don't think it's at that threshold yet.

Good call, removed the duplicate buffer config from `application-test`. I'm leaving the spend profile config in `application-connected-test` since that uses a separate test-specific spend profile.

Reply:
I don't think we have have a standard term for CWB vs. non-CWB. I've generally thought of it as "current CWB vs non-CWB" or "Rawls vs. MC Terra"

Reply:
Thanks, that sounds good to me.
Leaving the application-connected-test seems ok for now, but I think it would also be ok to move those tests using the same spend profile all the rest of the tests are using. I'll leave it up to you if that's easy to do now or we should wait on.

Reply:
Good point, we're already using `wm-default-spend-profile` for client tests. Switched connected-tests over